tests/models/align/test_modeling_align.py

```diff
@@ -167,6 +167,10 @@ def create_and_test_config_common_properties(self):
     def test_inputs_embeds(self):
         pass
 
+    @unittest.skip(reason="AlignVisionModel does not use inputs_embeds")
+    def test_inputs_embeds_matches_input_ids(self):
+        pass
```

Comment:
Instead of a per-model exception, I'd suggest adding something like this in the test:

```py
signature = inspect.signature(model.forward)
if "inputs_embeds" not in signature:
    self.skipTest("model does not accept `inputs_embeds`")
```

This way we don't need many test overwrites 😉

Reply:
Yes, that is already in the test, I just did not remove yet the skips. Some models simply do not have the attribute get_inputs_embeddings() and yeah, I see your suggestion below. That can work :)